sdk/eventhub/Azure.Messaging.EventHubs/Azure.Messaging.EventHubs.sln

```diff
@@ -9,8 +9,6 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.Messaging.EventHubs.T
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.Messaging.EventHubs.Samples", "samples\Azure.Messaging.EventHubs.Samples.csproj", "{AD33C619-AC51-4F29-937B-184B4F785F57}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.Core", "..\..\core\Azure.Core\src\Azure.Core.csproj", "{6864EFDA-9CE3-4F93-8060-554019389767}"
```

Comment:
Note:  This does not remove the dependency or reference to Core, just removes it as a member of the Event Hubs solution-space.